packages/dx-react-chart/docs/guides/animation.md

```diff
@@ -1,5 +1,24 @@
 # React Chart - Animation
 
-You can add the [Animation](../reference/animation.md) plugin to the Chart component to animate all the series when they first appear on-screen. To modify animation options, implement a custom point component for a series. In the following example, the duration of each bar's animation is configured individually:
+To animate chart series, add the [Animation](../reference/animation.md) plugin to the Chart component.
 
-.embedded-demo({ "path": "chart-basic/animation", "showThemeSelector": true })
+## Basic Setup
+
+Import the [Animation](../reference/animation.md) plugin.
+
+.embedded-demo({ "path": "animation/simple-animation", "showThemeSelector": true })
+
+
+## Animation on Transitions
+
+The following example shows how series are animated on data update:
+
+.embedded-demo({ "path": "animation/with-transition", "showThemeSelector": true })
+
+## Customize Animation
+
+Use the `Animation` plugin's `duration` and `easing` properties to specify animation's duration and easing type.
```

Comment:
"to specify animation's duration" a? the?

Reply:
"Animation" is uncountable, so neither.

Reply:
In this context, it is not. "Can I have some milk?" uncountable - no article. "The milk's colour is strange", not "milk's colour is strange". It is a possessive noun, not an uncountable noun. So, it is not neither.

Reply:
Okay, then it shouldn't be a possessive noun.

```suggestion
Use the `Animation` plugin's `duration` and `easing` properties to specify animation duration and easing type.
```

Reply:
Roman, 'animation', like 'text', depends on the context - they can be countable or uncountable. In this case, it should be: 
```suggestion
Use the `Animation` plugin's `duration` and `easing` properties to specify the animation's duration and easing type.
```

Reply:
Or
```suggestion
Use the `Animation` plugin's `duration` and `easing` properties to specify an animation's duration and easing type.
```

Reply:
an‧i‧ma‧tion /ˌænəˈmeɪʃən/ noun    
1 [uncountable] the process of making animated films, television programs, computer games etc
 They used computer animation in the film.
2 [countable] a film, television program, computer game etc that has pictures, clay models etc that seem to be really moving